lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.html

```diff
@@ -41,6 +41,7 @@
                             </mat-autocomplete>
                             <button
                                 id="adf-select-process-dropdown"
+                                title="{{ 'ADF_CLOUD_PROCESS_LIST.ADF_CLOUD_START_PROCESS.FORM.SELECT_PROCESS_DROPDOWN' | translate }}"
```

Comment:
If we don't specifically need the tooltip to appear please use aria-label instead, this should have no impact on UI

Reply:
changed to aria-label

Reply:
removing title means there won't be a tooltip anymore